README.md

```diff
@@ -1,20 +1,5 @@
-# webpki-roots
-This is a crate containing Mozilla's root certificates for use with
+This workspace contains the crate webpki-roots.
+
+The webpki-roots crate contains Mozilla's root certificates for use with
 the [webpki](https://github.com/rustls/webpki) or
 [rustls](https://github.com/rustls/rustls) crates.
-
-This crate is inspired by [certifi.io](https://certifi.io/en/latest/) and
```

Comment:
This is removed only to be brought back (in slightly modified form) in the next commit. Let's leave it here in the first place? Sorry, missed this in the previous round due to the other things.

Reply:
I've made the change to keep the paragraph that starts with 'The webpki-roots crate is inspired...' and all the paragraphs below it into the top-level README.md. Did you want only the paragraph 'The webpki-roots crate is inspired by...' to remain in the top-level README.md or that paragraph and the ones below it as well?